--- kbilliards-0.8.7b/admin/cvs.sh.orig	2008-09-01 00:07:42.000000000 +0200
+++ kbilliards-0.8.7b/admin/cvs.sh	2008-09-01 00:13:50.000000000 +0200
@@ -64,7 +64,7 @@
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* | automake*1.10*) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
